lib/looker-sdk/client.rb

```diff
@@ -396,14 +397,16 @@ def build_response(connection, request)
           http.read_timeout = connection.options.timeout rescue 60
 
           http.request(http_request) do |response|
+            if response.code != "200"
```

Comment:
oh yeah, there are other valid non-200 error codes (2xx) that could be received from the API that aren't an error.

Reply:
200 OK
201 Created
202 Accepted
203 Non-Authoritative Information
204 No Content
205 Reset Content
206 Partial Content
207 Multi-Status
208 Already Reported
226 IM Used